src/main/java/ch/njol/skript/effects/EffIgnite.java

```diff
@@ -36,72 +36,72 @@
 import ch.njol.skript.util.Timespan;
 import ch.njol.util.Kleenean;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Ignite/Extinguish")
-@Description({"Lights entities on fire or extinguishes them."})
-@Examples({"ignite the player",
-		"extinguish the player"})
+@Description("Lights entities on fire or extinguishes them.")
+@Examples({
+	"ignite the player",
+	"extinguish the player"
+})
 @Since("1.4")
 public class EffIgnite extends Effect {
+
 	static {
 		Skript.registerEffect(EffIgnite.class,
 				"(ignite|set fire to) %entities% [for %-timespan%]", "(set|light) %entities% on fire [for %-timespan%]",
 				"extinguish %entities%");
 	}
-	
-	private final static int DEFAULT_DURATION = 8 * 20; // default is 8 seconds for lava and fire, I didn't test other sources
-	
-	@SuppressWarnings("null")
+
+	private final static int DEFAULT_DURATION = 8 * 20; // default is 8 seconds for lava and fire.
```

Comment:
Source? https://minecraft.fandom.com/wiki/Lava#Burning says 15 seconds for lava, not saying that we should change this.

Reply:
Might actually be out of scope for this PR but worth looking at / just removing the comment:
```suggestion
	private static final int DEFAULT_DURATION = 8 * 20;
```

Reply:
Lava is different from fire

Reply:
Hence the comment is incorrect